src/AppInstallerCLICore/Command.cpp

```diff
@@ -489,4 +512,14 @@ namespace AppInstaller::CLI
         context.Reporter.Error() << Resource::String::PendingWorkError << std::endl;
         THROW_HR(E_NOTIMPL);
     }
+
+    void Command::OutputEnableMessage(Execution::Context& context) const
+    {
+        THROW_HR_IF(E_UNEXPECTED, m_feature == ExperimentalFeature::None);
+
+        auto featureInfo = UserSettings::GetFeatureInfo(m_feature);
+        context.Reporter.Warn() << Resource::String::CommandDisabled << " " << m_name << std::endl;
+        context.Reporter.Warn() << Resource::String::ExperimentalFeatureName << " " << std::get<0>(featureInfo) << std::endl;

```

Comment:
>context.Reporter.Warn() [](start = 8, length = 23)

No need to get a new output object, just use:

Warn() <<
    stuff << endl <<
    more stuff << endl;

Also, it probably is not a great idea to try and inject multiple strings into the middle of localized text like this.  Much better to have a single localized string and value after it.